application/src/main/java/org/togetherjava/tjbot/commands/basic/SuggestionsUpDownVoter.java

```diff
@@ -56,7 +58,15 @@ private static void reactWith(@NotNull String emoteName, @NotNull String fallbac
                     "Unable to vote on a suggestion with the configured emote ('{}'), using fallback instead.",
                     emoteName);
             return message.addReaction(fallbackUnicodeEmote);
-        }).queue();
+        }).queue(ignored -> {
+        }, exception -> {
+            if (exception instanceof ErrorResponseException && ((ErrorResponseException) exception)
+                .getErrorResponse() == ErrorResponse.REACTION_BLOCKED) {
+                return;
+            }
+
+            logger.error(exception.getMessage(), exception);
```

Comment:
Just log the exception, it'll print the exception message twice now

Reply:
to add to what tais says, the loggers `logger.foo(..., exception)` already appends the full error message, stack trace whatsoever to the log. the first part is just ur text of whats going on and to provide context. like `logger.error("Attempted to react to a suggestion, but failed for an unknown reason", exception);`

Reply:
leads me into thinking that u actually didnt try out the code to see how it looks, but just coded and pushed it right of the bat. u shouldnt do that, trigger ur new code to verify it.

Reply:
I am pretty sure it won't. how do you suggest I log it?

Reply:
> I am pretty sure it won't.

I am pretty sure it does. Right now it will say

> Foo bar
> FooException: Foo bar
> at ...
> at ...

So the message is twice. Just completely unecessary. Instead, add context.

> how do you suggest I log it?

Just like I showed above. So
```java
logger.error("Attempted to react to a suggestion, but failed for an unknown reason", exception);
```

Reply:
Dont assume. Try it out and you will see yourself.

Reply:
I just tested it by removing the `if` statement completely, blocking the bot and sending a message, and I got this:
https://gist.github.com/MaiTheLord/90c678e4948d733c7a4edcb7a48dd715
it sent it twice as it tries to send two embeds.

Reply:
```
22:01:21.247 [ForkJoinPool.commonPool-worker-1] ERROR org.togetherjava.tjbot.commands.basic.SuggestionsUpDownVoter - 90001: Reaction blocked
net.dv8tion.jda.api.exceptions.ErrorResponseException: 90001: Reaction blocked
```

Reply:
When it logs an exception the stack trace shows this.

![image](https://user-images.githubusercontent.com/63251610/158253657-aac40b53-9635-4d8c-85a1-bf22025e7d26.png)

is that supposed to happen? it doesn't happen if I don't use any error handler for the `queue()`.

Reply:
this solves it, I guess. is that good?
```java
logger.error("", exception);
```

Reply:
> leads me into thinking that u actually didnt try out the code to see how it looks, but just coded and pushed it right of the bat. u shouldnt do that, trigger ur new code to verify it.

actually I did test it. I just didn't pay much attention to the 2 first line of the stacktrace, unfortunately,

Reply:
> ```java
> logger.error("Attempted to react to a suggestion, but failed for an unknown reason", exception);
> ```

I wouldn't say "unknown reason", as the reason is stated in the stacktrace. maybe just
```java
logger.error("Attempted to react to a suggestion, but failed", exception);
```

Reply:
sounds good

Reply:
can I resolve this?

Reply:
yes